### ISSUE TITLE
Unexpected behavior when geojsonIdKey values of string type

### ISSUE THREAD

**Issue by MichaelCouch** [voluntadpear/vue-choropleth#13]
I'm having a problem colouring a map where the values for geojsonIdKey and the (data) idKey are strings (rather than integers). 

In src/components/ChoroplethLayer.vue the assumption there is an assumption that keys are numbers, and this appears deliberate.  For example I see 
```
let itemGeoJSONID = Number(feature.properties[this.geojsonIdKey])
```

for instance, in:

```
  data() {
    return {
      currentItem: { name: "", value: 0 },
      geojsonOptions: {
        style: feature => {
          let itemGeoJSONID = Number(feature.properties[this.geojsonIdKey])
          let color = "NONE"
          const {data} = this.geojsonData
          let item = data.find(x => x[this.idKey] === itemGeoJSONID)
          if (!item) {
            return {
              color: "white",
              weight: this.strokeWidth
            }
          }
...
```
Because of this recasting of the geojsonIdKey as a number, the following externally sourced geojson/data combination doesn't work:
geojson structured like
```
geojson.features[0].properties.geojsonIdKey = "SA4101"
geojson.features[1].properties.geojsonIdKey = "SA4102"
geojson.features[2].properties.geojsonIdKey = "SA4103"
...en
```
and data like
```
[
  {idKey: "SA4101", value: 77},
  {idKey: "SA4102", value: 71},
  {idKey: "SA4103", value: 81},
  ....
]
```

manually replacing these keys with alternative numerical fixes the issue, but is it actually necessary to make the assumption that geojsonIdKey and idKey are numbers?

**Comment by spiderpug** [comment]
I actually encountered the same problem today. I can't see why this shouldn't work with non-numeric keys. We are also using data objects that contain ISO codes instead of numbers:

```js
[
  { id: "CA", value: 13 },
  { id: "ES", value: 1 },
  ...
]
```

**Comment by brunosfreitas** [comment]
I'm also dealing with this problem here. 

I got the countries outline polygon from: https://github.com/johan/world.geo.json/blob/master/countries.geo.json?short_path=afdfc39

and there the unique data bellow properties is "name": "Country name as string". Should i add a new numeric key for each country to count as the intended link?

**Comment by voluntadpear** [comment]
This was fixed on version `0.3.1`. Can you try again? @MichaelCouch
